mlops/parallelm/mlops/ml_metrics_stat/regression/regression_stat_object_factory.py

```diff
@@ -35,18 +35,26 @@ def get_mlops_explained_variance_score_stat_object(**kwargs):
     @staticmethod
     def get_mlops_mean_absolute_error_stat_object(**kwargs):
         """
-        Method will create MLOps Single value stat object from numeric real number - mean absolute error
-        It is not recommended to access this method without understanding single value data structure that it is returning.
-        :param kwargs: mean absolute error
-        :return: Single Value stat object which has mean absolute error embedded inside
+        Method will create MLOps Single/Multiline value stat object from numeric real number - mean absolute error; or list of errors
```

Comment:
When I used will, @DrewParallel said present should be used: Method creates

Reply:
Yup